app/Http/Controllers/RoleController.php

```diff
@@ -0,0 +1,112 @@
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $roles = Role::all();
+
+        return view('manage.roles.index')->withRoles($roles);
+    }
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $permissions = Permission::all();
+
+        return view('manage.roles.create')->withPermissions($permissions);
+    }
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(CreateRolerRequests $request)
+    {
+        $role = new Role();
+        $role->display_name = $request->display_name;
+        $role->name = $request->name;
+        $role->description = $request->description;
+        $role->save();
+        if ($request->permissions) {
+            $role->syncPermissions(explode(',', $request->permissions));
+        }
+        Session::flash('success', 'Successfully created the new ' . $role->display_name . ' role in the database.');
+
+        return redirect()->route('roles.show', $role->id);
+    }
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $role = Role::where('id', $id)->with('permissions')->first();
+
+        return view('manage.roles.show')->withRole($role);
+    }
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $role = Role::where('id', $id)->with('permissions')->first();
+        $permissions = Permission::all();
+
+        return view('manage.roles.edit')->withRole($role)->withPermissions($permissions);
+    }
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function update(UpdateRoleRequest $request, $id)
+    {
+        $this->validateWith(['display_name' => 'required|max:255', 'description' => 'sometimes|max:255']);
+        $role = Role::findOrFail($id);
+        $role->display_name = $request->display_name;
+        $role->description = $request->description;
+        $role->save();
+        if ($request->permissions) {
+            $role->syncPermissions($request->permissions);
+        }
+        Session::flash('success', 'Successfully update the ' . $role->display_name . ' role in the database.');
+
+        return redirect()->route('roles.show', $id);
+    }
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id) {
+        //
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/RoleController.php</br>**Line**: 111</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace